docs/book/src/user/quick-start.md

```diff
@@ -731,7 +731,19 @@ spec:
 {{#/tab }}
 {{#/tabs }}
 
-After the controlplane is up and running, let's retrieve the [target cluster] Kubeconfig:
+To verify the control plane is up, check if the control plane machine
+has a ProviderID.
+```
+kubectl get machines --selector cluster.x-k8s.io/control-plane
+```
+
+If the control plane is deployed via [KubeadmControlPlane
+controller][control-plane-controller] ensure the control plane is ready.
+```
+kubectl get kubeadmcontrolplane --output jsonpath="{range .items[*]} [{.metadata.name} {.status.Ready}] {end}"
+```
+
+After the control plane node is up, we can retrieve the workload cluster Kubeconfig:
```

Comment:
For v1a3 I'm using the terminology of `workload cluster` rather than `target cluster`.